### DISCUSSION TITLE
1.1.6

### DISCUSSION THREAD

**Discussion by benhillis** [microsoft/WSL#9874] [General]
*  Add a pipeline step to validate localization changes. This will prevent crashes for non-English locales when translation errors occur (solves #9796)
*  Cleanup the winrt factory cache before calling CoUninitialize() to prevent crashes.
*  Flush disk before detaching it from the guest.
*  Introduce wslrelay.exe which replaces wslhost.exe for some operations.
*  Move declaration of service helpers next to where they are invoked (solves #9781).

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/WSL/releases/tag/1.1.6'>1.1.6</a>.</em>

**Comment by cmalcolmch** [comment]
Hi 

I was on version 1.1.3

I just typed wsl --update

I have ended up on 1.1.6 🤷‍♀️

Checking the release page.

1.1.3 Latest
1.1.6 Pre-release

> wsl --version
WSL version: 1.1.6.0
Kernel version: 5.15.90.1
WSLg version: 1.0.50
MSRDC version: 1.2.3770
Direct3D version: 1.608.2-61064218
DXCore version: 10.0.25131.1002-220531-1700.rs-onecore-base2-hyp
Windows version: 10.0.19045.2728

I thought if I was on 1.1.3 I would stay on 1.1.3 as it's the latest release.

Happy to stay on 1.1.6, but curious as to what's just happened, as it's not the behavior I would have expected.

**Reply by roelarents** [reply]
Maybe a timing issue. I think it was a regular release before. Probably made into a pre-release because of problems people are experiencing with docker and v2 cgroups.

**Reply by MorphBonehunter** [reply]
Details could be found in https://github.com/microsoft/WSL/issues/9866#issuecomment-1490707255
